Fix API docs (#310)
auth0/auth0-oidc-client-net: CONTRIBUTING.md

- Maintain the existing minumum .NET framework/core support.
- Keep PRs focused and change the minimum number of lines to achieve your goal.
- Do not introduce breaking changes without prior discussion and approval.

### API documentation

> As the API documentation requires a windows environment, it is built and deployed by running the correspondig GitHub Actions Workflow.

Author: ```suggestion
As the API documentation requires a windows environment, it is built and deployed automatically when running the release GitHub Actions workflow.
```
